### PR TITLE
Development

### PR DESCRIPTION
**Pull request by emmanuelkamala**

### CODE REVIEW

**Comment by mcrd25** [comment]
# Currently under review by TSE [Maya](https://twitter.com/mcrd25)

File: Tutorial_9_Self_join.sql [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+-- Question 9
+SELECT DISTINCT stopb.name, a.company, a.num
+FROM route AS a
+INNER JOIN route AS b
+ON (a.company = b.company AND a.num = b.num)
+INNER JOIN stops AS stopa ON a.stop = stopa.id
+INNER JOIN stops AS stopb ON b.stop = stopb.id
+WHERE stopa.name ='Craiglockhart'
+AND a.company ='LRT'
+
```
Comment (mcrd25) [inline Tutorial_9_Self_join.sql:71]: You forgot question 10

**Review by mcrd25 — APPROVED** [review]
# Everything is Great! You may 🚀 

Hi @emmanuelkamala 👋,

**You did not actually have to submit a code review for this project since you get immediate feedback, however, I have checked everything and they all are correct! Congratulations on completing everything in this section. If you know anyone yet to submit this project please inform them that submitting code review request is not necessary.
Please do question 10 of Self Join before submitting project completion**

***Congratulations on completing SQL Section. Next up, RAILS! YAY 🎊 🎉 🥇***

***If you have any questions, now or in the future, do not hesitate to contact me on social media or on slack and I will be glad to chat with you.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/)

File: Tutorial_9_Self_join.sql (hunk 1 above)
Comment (emmanuelkamala) [reply Tutorial_9_Self_join.sql:71]: haha, Okay, I forgot that instruction of not requesting a review, even logically it does not make sense to submit for a review, but on the other hand, I am glad, I did, cause I could have submitted it without the question number 10 being included. Thank you, let me do that and submit the project completion form

**Pull request merged**